### DISCUSSION TITLE
Group of Lot with only one Lot in notice 16

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#670] [Q&A] [answered]
Hi Lab
I sent a notice type 16 and it was submitted without problems. SDK 1.8
In this notice there is only one lot but in doc https://docs.ted.europa.eu/eforms/latest/schema/procedure-lot-part-information.html 
it is written : 
They shall not contain less than two Lots;
My question : 
That is a sort of 'recommendation' ? Or there is another reason ?
The notice : 
`<?xml version="1.0" encoding="UTF-8"?><ContractNotice xmlns="urn:oasis:names:specification:ubl:schema:xsd:ContractNotice-2" xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2" xmlns:efac="http://data.europa.eu/p27/eforms-ubl-extension-aggregate-components/1" xmlns:efbc="http://data.europa.eu/p27/eforms-ubl-extension-basic-components/1" xmlns:efext="http://data.europa.eu/p27/eforms-ubl-extensions/1" xmlns:ext="urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2">
	<ext:UBLExtensions xmlns="">
		<ext:UBLExtension>
			<ext:ExtensionContent>
				<efext:EformsExtension>
					<efac:NoticeSubType>
						<cbc:SubTypeCode listName="notice-subtype">16</cbc:SubTypeCode>
					</efac:NoticeSubType>
					<efac:Organizations>
						<efac:Organization>
							<efac:Company>
								<cac:PartyIdentification>
									<cbc:ID>ORG-0001</cbc:ID>
								</cac:PartyIdentification>
								<cac:PartyName>
									<cbc:Name languageID="ITA">Official name(BT-500-ORGANIZATION-COMPANY) 1</cbc:Name>
								</cac:PartyName>
								<cac:PostalAddress>
									<cbc:Department>Department(BT-16-ORGANIZATION-COMPANY) 1</cbc:Department>
									<cbc:CityName>Town(BT-513-ORGANIZATION-COMPANY) 1</cbc:CityName>
									<cac:Country>
										<cbc:IdentificationCode listName="country">ABW</cbc:IdentificationCode>
									</cac:Country>
								</cac:PostalAddress>
								<cac:PartyLegalEntity>
									<cbc:CompanyID>2012</cbc:CompanyID>
								</cac:PartyLegalEntity>
								<cac:Contact>
									<cbc:Telephone>232323</cbc:Telephone>
									<cbc:Telefax>2323</cbc:Telefax>
									<cbc:ElectronicMail>BT-506-ORGANIZATION-COMPANY1@gmail.com</cbc:ElectronicMail>
								</cac:Contact>
							</efac:Company>
						</efac:Organization>
						<efac:Organization>
							<efac:Company>
								<cac:PartyIdentification>
									<cbc:ID>ORG-0002</cbc:ID>
								</cac:PartyIdentification>
								<cac:PartyName>
									<cbc:Name languageID="ITA">Official name(BT-500-ORGANIZATION-COMPANY) 2</cbc:Name>
								</cac:PartyName>
								<cac:PostalAddress>
									<cbc:Department>Department(BT-16-ORGANIZATION-COMPANY) 2</cbc:Department>
									<cbc:CityName>Town(BT-513-ORGANIZATION-COMPANY) 2</cbc:CityName>
									<cac:Country>
										<cbc:IdentificationCode listName="country">ITA</cbc:IdentificationCode>
									</cac:Country>
								</cac:PostalAddress>
								<cac:PartyLegalEntity>
									<cbc:CompanyID>2013</cbc:CompanyID>
								</cac:PartyLegalEntity>
								<cac:Contact>
									<cbc:Telephone>99998888</cbc:Telephone>
									<cbc:Telefax>9988</cbc:Telefax>
									<cbc:ElectronicMail>BT-506-ORGANIZATION-COMPANY2@gmail.com</cbc:ElectronicMail>
								</cac:Contact>
							</efac:Company>
						</efac:Organization>
					</efac:Organizations>
				</efext:EformsExtension>
			</ext:ExtensionContent>
		</ext:UBLExtension>
	</ext:UBLExtensions>
	<cbc:UBLVersionID xmlns="">2.3</cbc:UBLVersionID>
	<cbc:CustomizationID xmlns="">eforms-sdk-1.8</cbc:CustomizationID>
	<cbc:ID schemeName="notice-id" xmlns="">e2c4de45-816e-46e6-b840-ab5c19562894</cbc:ID>
	<cbc:ContractFolderID xmlns="">721c19a0-727e-440c-b40a-2ef7e0963ed9</cbc:ContractFolderID>
	<cbc:IssueDate xmlns="">2023-09-14Z</cbc:IssueDate>
	<cbc:IssueTime xmlns="">09:41:32Z</cbc:IssueTime>
	<cbc:VersionID xmlns="">01</cbc:VersionID>
	<cbc:RegulatoryDomain xmlns="">31985R2137</cbc:RegulatoryDomain>
	<cbc:NoticeTypeCode listName="competition" xmlns="">cn-standard</cbc:NoticeTypeCode>
	<cbc:NoticeLanguageCode listName="language" xmlns="">ITA</cbc:NoticeLanguageCode>
	<cac:ContractingParty xmlns="">
		<cbc:BuyerProfileURI>http://bt508ProcedureBuyer.com</cbc:BuyerProfileURI>
		<cac:ContractingPartyType>
			<cbc:PartyTypeCode listName="buyer-legal-type">body-pl</cbc:PartyTypeCode>
		</cac:ContractingPartyType>
		<cac:ContractingActivity>
			<cbc:ActivityTypeCode listName="authority-activity">gen-pub</cbc:ActivityTypeCode>
		</cac:ContractingActivity>
		<cac:Party>
			<cac:PartyIdentification>
				<cbc:ID>ORG-0001</cbc:ID>
			</cac:PartyIdentification>
			<cac:ServiceProviderParty>
				<cbc:ServiceTypeCode listName="organisation-role">serv-prov</cbc:ServiceTypeCode>
				<cac:Party>
					<cac:PartyIdentification>
						<cbc:ID>ORG-0002</cbc:ID>
					</cac:PartyIdentification>
				</cac:Party>
			</cac:ServiceProviderParty>
		</cac:Party>
	</cac:ContractingParty>
	<cac:TenderingTerms xmlns="">
		<cac:ProcurementLegislationDocumentReference>
			<cbc:ID>Legal basis of the procedure (ELI - CELEX)(bt_01_c__procedure)</cbc:ID>
			<cbc:DocumentDescription languageID="ITA">Legal basis of the procedure (description)(bt_01_d__procedure)</cbc:DocumentDescription>
		</cac:ProcurementLegislationDocumentReference>
		<cac:ProcurementLegislationDocumentReference>
			<cbc:ID>LocalLegalBasis</cbc:ID>
			<cbc:DocumentDescription languageID="ITA">Legal basis(bt_01_f__procedure)</cbc:DocumentDescription>
		</cac:ProcurementLegislationDocumentReference>
		<cac:ProcurementLegislationDocumentReference>
			<cbc:ID>CrossBorderLaw</cbc:ID>
			<cbc:DocumentDescription languageID="ITA">Description(bt_09_b__procedure)</cbc:DocumentDescription>
		</cac:ProcurementLegislationDocumentReference>
		<cac:TendererQualificationRequest>
			<cac:SpecificTendererRequirement>
				<cbc:TendererRequirementTypeCode listName="exclusion-ground">bankr-nat</cbc:TendererRequirementTypeCode>
				<cbc:Description languageID="ITA">Description(bt_67_b__procedure)</cbc:Description>
			</cac:SpecificTendererRequirement>
		</cac:TendererQualificationRequest>
		<cac:LotDistribution>
			<cbc:MaximumLotsAwardedNumeric>1</cbc:MaximumLotsAwardedNumeric>
			<cbc:MaximumLotsSubmittedNumeric>1</cbc:MaximumLotsSubmittedNumeric>
			<cac:LotsGroup>
				<cbc:LotsGroupID>GLO-0001</cbc:LotsGroupID>
				<cac:ProcurementProjectLotReference>
					<cbc:ID schemeName="Lot">LOT-0001</cbc:ID>
				</cac:ProcurementProjectLotReference>
				<cac:ProcurementProjectLotReference>
					<cbc:ID schemeName="Lot">LOT-0001</cbc:ID>
				</cac:ProcurementProjectLotReference>
			</cac:LotsGroup>
		</cac:LotDistribution>
	</cac:TenderingTerms>
	<cac:TenderingProcess xmlns="">
		<cbc:Description languageID="ITA">Main features of the procedure(bt_88_procedure)</cbc:Description>
		<cbc:ProcedureCode listName="procurement-procedure-type">open</cbc:ProcedureCode>
		<cac:NoticeDocumentReference>
			<cbc:ID>eeecf7d8-843a-4294-b7e8-4835e2161386-99</cbc:ID>
		</cac:NoticeDocumentReference>
		<cac:ProcessJustification>
			<cbc:ProcessReasonCode listName="accelerated-procedure">true</cbc:ProcessReasonCode>
			<cbc:ProcessReason languageID="ITA">Justification for the accelerated procedure(bt_1351_procedure)</cbc:ProcessReason>
		</cac:ProcessJustification>
	</cac:TenderingProcess>
	<cac:ProcurementProject xmlns="">
		<cbc:ID>Internal identifier(bt_22_procedure)</cbc:ID>
		<cbc:Name languageID="ITA">Title(bt_21_procedure)</cbc:Name>
		<cbc:Description languageID="ITA">Description(bt_24_procedure)</cbc:Description>
		<cbc:ProcurementTypeCode listName="contract-nature">services</cbc:ProcurementTypeCode>
		<cbc:Note languageID="ITA">Additional information(bt_300_procedure)</cbc:Note>
		<cac:RequestedTenderTotal>
			<ext:UBLExtensions>
				<ext:UBLExtension>
					<ext:ExtensionContent>
						<efext:EformsExtension>
							<efbc:FrameworkMaximumAmount currencyID="EUR">50</efbc:FrameworkMaximumAmount>
						</efext:EformsExtension>
					</ext:ExtensionContent>
				</ext:UBLExtension>
			</ext:UBLExtensions>
			<cbc:EstimatedOverallContractAmount currencyID="EUR">100</cbc:EstimatedOverallContractAmount>
		</cac:RequestedTenderTotal>
		<cac:MainCommodityClassification>
			<cbc:ItemClassificationCode listName="cpv">80110000</cbc:ItemClassificationCode>
		</cac:MainCommodityClassification>
		<cac:RealizedLocation>
			<cbc:Description languageID="ITA">Additional information(bt_728_procedure)</cbc:Description>
			<cac:Address>
				<cbc:StreetName>Postal address(bt_5101_a__procedure)</cbc:StreetName>
				<cbc:AdditionalStreetName>Postal address(bt_5101_b__procedure)</cbc:AdditionalStreetName>
				<cbc:CityName>Town(bt_5131_procedure)</cbc:CityName>
				<cbc:PostalZone>Postcode(bt_5121_procedure)</cbc:PostalZone>
				<cbc:CountrySubentityCode listName="nuts">AL013</cbc:CountrySubentityCode>
				<cac:AddressLine>
					<cbc:Line>Postal address(bt_5101_c__procedure)</cbc:Line>
				</cac:AddressLine>
				<cac:Country>
					<cbc:IdentificationCode listName="country">BEL</cbc:IdentificationCode>
				</cac:Country>
			</cac:Address>
		</cac:RealizedLocation>
	</cac:ProcurementProject>
	<cac:ProcurementProjectLot xmlns="">
		<cbc:ID schemeName="Lot">LOT-0001</cbc:ID>
		<cac:TenderingTerms>
			<ext:UBLExtensions>
				<ext:UBLExtension>
					<ext:ExtensionContent>
						<efext:EformsExtension>
							<efac:SelectionCriteria>
								<cbc:CriterionTypeCode listName="selection-criterion">ef-stand</cbc:CriterionTypeCode>
								<cbc:Name languageID="ITA">Name(bt_749_lot)</cbc:Name>
								<cbc:Description languageID="ITA">Description(bt_750_lot)</cbc:Description>
								<efac:CriterionParameter>
									<efbc:ParameterNumeric>89</efbc:ParameterNumeric>
								</efac:CriterionParameter>
							</efac:SelectionCriteria>
						</efext:EformsExtension>
					</ext:ExtensionContent>
				</ext:UBLExtension>
			</ext:UBLExtensions>
			<cbc:FundingProgramCode listName="eu-funded">eu-funds</cbc:FundingProgramCode>
			<cbc:RecurringProcurementDescription languageID="ITA">Description(bt_95_lot)</cbc:RecurringProcurementDescription>
			<cbc:LatestSecurityClearanceDate>2023-09-09Z</cbc:LatestSecurityClearanceDate>
			<cac:RequiredFinancialGuarantee>
				<cbc:GuaranteeTypeCode listName="tender-guarantee-required">true</cbc:GuaranteeTypeCode>
				<cbc:Description languageID="ITA">Description of the financial guarantee(bt_75_lot)</cbc:Description>
			</cac:RequiredFinancialGuarantee>
			<cac:FiscalLegislationDocumentReference>
				<cbc:ID>ID – Tax legislation document(opt_111_lot_fiscallegis</cbc:ID>
				<cac:Attachment>
					<cac:ExternalReference>
						<cbc:URI>http://opt110LotFiscallegis.com</cbc:URI>
					</cac:ExternalReference>
				</cac:Attachment>
				<cac:IssuerParty>
					<cac:PartyIdentification>
						<cbc:ID>ORG-0001</cbc:ID>
					</cac:PartyIdentification>
				</cac:IssuerParty>
			</cac:FiscalLegislationDocumentReference>
			<cac:EnvironmentalLegislationDocumentReference>
				<cbc:ID>ID – Environmental legislation document(opt_112_lot_environlegis)</cbc:ID>
				<cac:Attachment>
					<cac:ExternalReference>
						<cbc:URI>http://opt112LotEnvironlegis.com</cbc:URI>
					</cac:ExternalReference>
				</cac:Attachment>
				<cac:IssuerParty>
					<cac:PartyIdentification>
						<cbc:ID>ORG-0002</cbc:ID>
					</cac:PartyIdentification>
				</cac:IssuerParty>
			</cac:EnvironmentalLegislationDocumentReference>
			<cac:EmploymentLegislationDocumentReference>
				<cbc:ID>INFORMATION ABOUT EMPLOYMENT LEGISLATION,ID – Employment legislation document(opt_113_lot_employlegis)</cbc:ID>
				<cac:Attachment>
					<cac:ExternalReference>
						<cbc:URI>http://opt130LotEmploylegis.com</cbc:URI>
					</cac:ExternalReference>
				</cac:Attachment>
				<cac:IssuerParty>
					<cac:PartyIdentification>
						<cbc:ID>ORG-0001</cbc:ID>
					</cac:PartyIdentification>
				</cac:IssuerParty>
			</cac:EmploymentLegislationDocumentReference>
			<cac:CallForTendersDocumentReference>
				<cbc:ID>ID – Procurement documents(opt_140_lot)</cbc:ID>
				<cbc:DocumentType>non-restricted-document</cbc:DocumentType>
				<cac:Attachment>
					<cac:ExternalReference>
						<cbc:URI>http://BT99LOT.com</cbc:URI>
					</cac:ExternalReference>
				</cac:Attachment>
			</cac:CallForTendersDocumentReference>
			<cac:PaymentTerms>
				<cbc:Note languageID="ITA">Financial arrangement(bt_77_lot)</cbc:Note>
			</cac:PaymentTerms>
			<cac:TendererQualificationRequest>
				<cbc:CompanyLegalFormCode listName="required">true</cbc:CompanyLegalFormCode>
				<cbc:CompanyLegalForm languageID="ITA">BT-76-LOT</cbc:CompanyLegalForm>
			</cac:TendererQualificationRequest>
			<cac:TendererQualificationRequest>
				<cac:SpecificTendererRequirement>
					<cbc:TendererRequirementTypeCode listName="missing-info-submission">late-all</cbc:TendererRequirementTypeCode>
					<cbc:Description languageID="ITA">BT-772-LOT-1</cbc:Description>
				</cac:SpecificTendererRequirement>
			</cac:TendererQualificationRequest>
			<cac:TendererQualificationRequest>
				<cac:SpecificTendererRequirement>
					<cbc:TendererRequirementTypeCode listName="reserved-procurement">res-pub-ser</cbc:TendererRequirementTypeCode>
				</cac:SpecificTendererRequirement>
			</cac:TendererQualificationRequest>
			<cac:ContractExecutionRequirement>
				<cbc:ExecutionRequirementCode listName="conditions">performance</cbc:ExecutionRequirementCode>
				<cbc:Description languageID="ITA">Conditions relating to the performance of the contract(bt_70_lot)</cbc:Description>
			</cac:ContractExecutionRequirement>
			<cac:ContractExecutionRequirement>
				<cbc:ExecutionRequirementCode listName="reserved-execution">yes</cbc:ExecutionRequirementCode>
			</cac:ContractExecutionRequirement>
			<cac:ContractExecutionRequirement>
				<cbc:ExecutionRequirementCode listName="einvoicing">required</cbc:ExecutionRequirementCode>
			</cac:ContractExecutionRequirement>
			<cac:ContractExecutionRequirement>
				<cbc:ExecutionRequirementCode listName="ecatalog-submission">not-allowed</cbc:ExecutionRequirementCode>
			</cac:ContractExecutionRequirement>
			<cac:ContractExecutionRequirement>
				<cbc:ExecutionRequirementCode listName="nda">true</cbc:ExecutionRequirementCode>
				<cbc:Description languageID="ITA">Additional information about the non-disclosure agreement(bt_802_lot)</cbc:Description>
			</cac:ContractExecutionRequirement>
			<cac:AwardingTerms>
				<cac:AwardingCriterion>
					<cbc:Description languageID="ITA">BT-733-LOT</cbc:Description>
					<cbc:CalculationExpression languageID="ITA">BT-543-LOT</cbc:CalculationExpression>
					<cac:SubordinateAwardingCriterion>
						<cbc:Name languageID="ITA">BT-734-LOT</cbc:Name>
						<cbc:Description languageID="ITA">(BT-540-LOT)</cbc:Description>
					</cac:SubordinateAwardingCriterion>
				</cac:AwardingCriterion>
			</cac:AwardingTerms>
			<cac:AdditionalInformationParty>
				<cac:PartyIdentification>
					<cbc:ID>ORG-0001</cbc:ID>
				</cac:PartyIdentification>
			</cac:AdditionalInformationParty>
			<cac:DocumentProviderParty>
				<cac:PartyIdentification>
					<cbc:ID>ORG-0002</cbc:ID>
				</cac:PartyIdentification>
			</cac:DocumentProviderParty>
			<cac:TenderRecipientParty>
				<cbc:EndpointID>http://www.bt18lot.com</cbc:EndpointID>
				<cac:PartyIdentification>
					<cbc:ID>ORG-0001</cbc:ID>
				</cac:PartyIdentification>
			</cac:TenderRecipientParty>
			<cac:TenderEvaluationParty>
				<cac:PartyIdentification>
					<cbc:ID>ORG-0002</cbc:ID>
				</cac:PartyIdentification>
			</cac:TenderEvaluationParty>
			<cac:TenderValidityPeriod>
				<cbc:DurationMeasure unitCode="DAY">89</cbc:DurationMeasure>
			</cac:TenderValidityPeriod>
			<cac:AppealTerms>
				<cac:PresentationPeriod>
					<cbc:Description languageID="ITA">Information about review deadlines(bt_99_lot)</cbc:Description>
				</cac:PresentationPeriod>
				<cac:AppealInformationParty>
					<cac:PartyIdentification>
						<cbc:ID>ORG-0002</cbc:ID>
					</cac:PartyIdentification>
				</cac:AppealInformationParty>
				<cac:AppealReceiverParty>
					<cac:PartyIdentification>
						<cbc:ID>ORG-0001</cbc:ID>
					</cac:PartyIdentification>
				</cac:AppealReceiverParty>
				<cac:MediationParty>
					<cac:PartyIdentification>
						<cbc:ID>ORG-0001</cbc:ID>
					</cac:PartyIdentification>
				</cac:MediationParty>
			</cac:AppealTerms>
			<cac:Language>
				<cbc:ID>ENG</cbc:ID>
			</cac:Language>
			<cac:PostAwardProcess>
				<cbc:ElectronicOrderUsageIndicator>true</cbc:ElectronicOrderUsageIndicator>
				<cbc:ElectronicPaymentUsageIndicator>true</cbc:ElectronicPaymentUsageIndicator>
			</cac:PostAwardProcess>
			<cac:SecurityClearanceTerm>
				<cbc:Code listName="required">true</cbc:Code>
				<cbc:Description languageID="ITA">BT-732-LOT</cbc:Description>
			</cac:SecurityClearanceTerm>
		</cac:TenderingTerms>
		<cac:TenderingProcess>
			<ext:UBLExtensions>
				<ext:UBLExtension>
					<ext:ExtensionContent>
						<efext:EformsExtension>
							<efbc:AccessToolName>Name(bt_632_lot)</efbc:AccessToolName>
						</efext:EformsExtension>
					</ext:ExtensionContent>
				</ext:UBLExtension>
			</ext:UBLExtensions>
			<cbc:SubmissionMethodCode listName="esubmission">allowed</cbc:SubmissionMethodCode>
			<cbc:GovernmentAgreementConstraintIndicator>true</cbc:GovernmentAgreementConstraintIndicator>
			<cbc:AccessToolsURI>http://bt124lot.com</cbc:AccessToolsURI>
			<cac:TenderSubmissionDeadlinePeriod>
				<cbc:EndDate>2023-09-15Z</cbc:EndDate>
				<cbc:EndTime>18:09:00Z</cbc:EndTime>
			</cac:TenderSubmissionDeadlinePeriod>
			<cac:NoticeDocumentReference>
				<cbc:ID>eeecf7d8-843a-4294-b7e8-4835e2161386-22</cbc:ID>
				<cbc:ReferencedDocumentInternalAddress>bt_1251_lot</cbc:ReferencedDocumentInternalAddress>
			</cac:NoticeDocumentReference>
			<cac:ProcessJustification>
				<cbc:ProcessReasonCode listName="no-esubmission-justification">ipr-iss</cbc:ProcessReasonCode>
				<cbc:Description languageID="ITA">Description(bt_745_lot)</cbc:Description>
			</cac:ProcessJustification>
			<cac:OpenTenderEvent>
				<cbc:OccurrenceDate>2023-09-15Z</cbc:OccurrenceDate>
				<cbc:OccurrenceTime>20:08:00Z</cbc:OccurrenceTime>
				<cbc:Description languageID="ITA">Additional information(bt_134_lot)</cbc:Description>
				<cac:OccurenceLocation>
					<cbc:Description languageID="ITA">Place(bt_133_lot)</cbc:Description>
				</cac:OccurenceLocation>
			</cac:OpenTenderEvent>
			<cac:AuctionTerms>
				<cbc:AuctionConstraintIndicator>true</cbc:AuctionConstraintIndicator>
				<cbc:Description languageID="ITA">Description(bt_122_lot)</cbc:Description>
				<cbc:AuctionURI>http://www.bt123lot.com</cbc:AuctionURI>
			</cac:AuctionTerms>
			<cac:FrameworkAgreement>
				<cbc:MaximumOperatorQuantity>2</cbc:MaximumOperatorQuantity>
				<cbc:Justification languageID="ITA">Justification for the duration of the framework agreement(bt_109_lot)</cbc:Justification>
				<cac:SubsequentProcessTenderRequirement>
					<cbc:Name>buyer-categories</cbc:Name>
					<cbc:Description languageID="ITA">Additional buyer coverage(bt_111_lot)</cbc:Description>
				</cac:SubsequentProcessTenderRequirement>
			</cac:FrameworkAgreement>
			<cac:ContractingSystem>
				<cbc:ContractingSystemTypeCode listName="framework-agreement">fa-mix</cbc:ContractingSystemTypeCode>
			</cac:ContractingSystem>
			<cac:ContractingSystem>
				<cbc:ContractingSystemTypeCode listName="dps-usage">dps-list</cbc:ContractingSystemTypeCode>
			</cac:ContractingSystem>
		</cac:TenderingProcess>
		<cac:ProcurementProject>
			<cbc:ID>Internal identifier(bt_22_lot)</cbc:ID>
			<cbc:Name languageID="ITA">Title(bt_21_lot)</cbc:Name>
			<cbc:Description languageID="ITA">Description(bt_24_lot)</cbc:Description>
			<cbc:ProcurementTypeCode listName="contract-nature">services</cbc:ProcurementTypeCode>
			<cbc:EstimatedOverallContractQuantity>20</cbc:EstimatedOverallContractQuantity>
			<cbc:Note languageID="ITA">Additional information(bt_300_lot)</cbc:Note>
			<cac:ProcurementAdditionalType>
				<cbc:ProcurementTypeCode listName="strategic-procurement">env-imp</cbc:ProcurementTypeCode>
				<cbc:ProcurementType languageID="ITA">Description(bt_777_lot)</cbc:ProcurementType>
			</cac:ProcurementAdditionalType>
			<cac:ProcurementAdditionalType>
				<cbc:ProcurementTypeCode listName="strategic-procurement">inn-pur</cbc:ProcurementTypeCode>
				<cbc:ProcurementType languageID="ITA">Description(bt_777_lot) 1 bis</cbc:ProcurementType>
			</cac:ProcurementAdditionalType>
			<cac:ProcurementAdditionalType>
				<cbc:ProcurementTypeCode listName="accessibility">inc</cbc:ProcurementTypeCode>
				<cbc:ProcurementType languageID="ITA">Justification(bt_755_lot)</cbc:ProcurementType>
			</cac:ProcurementAdditionalType>
			<cac:RequestedTenderTotal>
				<ext:UBLExtensions>
					<ext:UBLExtension>
						<ext:ExtensionContent>
							<efext:EformsExtension>
								<efbc:FrameworkMaximumAmount currencyID="EUR">12</efbc:FrameworkMaximumAmount>
							</efext:EformsExtension>
						</ext:ExtensionContent>
					</ext:UBLExtension>
				</ext:UBLExtensions>
				<cbc:EstimatedOverallContractAmount currencyID="EUR">100</cbc:EstimatedOverallContractAmount>
			</cac:RequestedTenderTotal>
			<cac:MainCommodityClassification>
				<cbc:ItemClassificationCode listName="cpv">75121000</cbc:ItemClassificationCode>
			</cac:MainCommodityClassification>
			<cac:RealizedLocation>
				<cbc:Description languageID="ITA">Additional information(bt_728_lot)</cbc:Description>
				<cac:Address>
					<cbc:StreetName>Postal address(bt_5101_a__lot)</cbc:StreetName>
					<cbc:AdditionalStreetName>Postal address(bt_5101_b__lot)</cbc:AdditionalStreetName>
					<cbc:CityName>Town(bt_5131_lot)</cbc:CityName>
					<cbc:PostalZone>Postcode(bt_5121_lot)</cbc:PostalZone>
					<cac:AddressLine>
						<cbc:Line>Postal address(bt_5101_c__lot)</cbc:Line>
					</cac:AddressLine>
					<cac:Country>
						<cbc:IdentificationCode listName="country">ARM</cbc:IdentificationCode>
					</cac:Country>
				</cac:Address>
			</cac:RealizedLocation>
			<cac:ContractExtension>
				<cbc:OptionsDescription languageID="ITA">The buyer reserves the right for additional purchases from the contractor, as described here(bt_54_lot)</cbc:OptionsDescription>
				<cbc:MaximumNumberNumeric>3</cbc:MaximumNumberNumeric>
			</cac:ContractExtension>
		</cac:ProcurementProject>
	</cac:ProcurementProjectLot>
	<cac:ProcurementProjectLot xmlns="">
		<cbc:ID schemeName="LotsGroup">GLO-0001</cbc:ID>
		<cac:TenderingTerms>
			<cac:AwardingTerms>
				<cac:AwardingCriterion>
					<cac:SubordinateAwardingCriterion>
						<ext:UBLExtensions>
							<ext:UBLExtension>
								<ext:ExtensionContent>
									<efext:EformsExtension>
										<efac:AwardCriterionParameter>
											<efbc:ParameterCode listName="number-weight">ord-imp</efbc:ParameterCode>
											<efbc:ParameterNumeric>1</efbc:ParameterNumeric>
										</efac:AwardCriterionParameter>
										<efac:AwardCriterionParameter>
											<efbc:ParameterCode listName="number-fixed">fix-tot</efbc:ParameterCode>
											<efbc:ParameterNumeric>50</efbc:ParameterNumeric>
										</efac:AwardCriterionParameter>
									</efext:EformsExtension>
								</ext:ExtensionContent>
							</ext:UBLExtension>
						</ext:UBLExtensions>
						<cbc:AwardingCriterionTypeCode listName="award-criterion-type">cost</cbc:AwardingCriterionTypeCode>
						<cbc:Description languageID="ITA">cost desc</cbc:Description>
					</cac:SubordinateAwardingCriterion>
				</cac:AwardingCriterion>
			</cac:AwardingTerms>
		</cac:TenderingTerms>
		<cac:TenderingProcess>
			<cac:FrameworkAgreement>
				<cbc:EstimatedMaximumValueAmount currencyID="EUR">22</cbc:EstimatedMaximumValueAmount>
			</cac:FrameworkAgreement>
		</cac:TenderingProcess>
		<cac:ProcurementProject>
			<cbc:ID>IDGROUPLOTS1</cbc:ID>
			<cbc:Name languageID="ITA">title group of lots</cbc:Name>
			<cbc:Description languageID="ITA">desc group of lots</cbc:Description>
			<cac:RequestedTenderTotal>
				<ext:UBLExtensions>
					<ext:UBLExtension>
						<ext:ExtensionContent>
							<efext:EformsExtension>
								<efbc:FrameworkMaximumAmount currencyID="EUR">23</efbc:FrameworkMaximumAmount>
							</efext:EformsExtension>
						</ext:ExtensionContent>
					</ext:UBLExtension>
				</ext:UBLExtensions>
				<cbc:EstimatedOverallContractAmount currencyID="EUR">23</cbc:EstimatedOverallContractAmount>
			</cac:RequestedTenderTotal>
		</cac:ProcurementProject>
	</cac:ProcurementProjectLot>
</ContractNotice>
`
Best Regards
Renato

**Comment by YvesJo** [comment, marked as answer]
Hi,
To form a group, there is the need for two different lots.
There are two references to a same lot in the instance. The uniqueness value control from another level than root is pending the next EFX language version (i.e. EFX 2).
